rotkehlchen/api/rest.py

```diff
@@ -3207,6 +3207,20 @@ def upload_asset_icon(self, asset: Asset, filepath: Path) -> Response:
             status_code=HTTPStatus.OK,
         )
 
+    def refresh_asset_icon(self, asset: Asset) -> Response:
+        """Deletes an asset's icon from the cache and requeries it."""
+        is_deleted, message = self.rotkehlchen.icon_manager.delete_icon(asset)
+        if not is_deleted:
```

Comment:
explicity truthy checks

```suggestion
        if is_deleted is False:
```